app/build.gradle

```diff
@@ -166,7 +166,8 @@ dependencies {
     implementation 'com.github.deano2390:MaterialShowcaseView:1.2.0@aar'
     implementation "com.jakewharton:butterknife:$rootProject.butterknifeVersion"
     annotationProcessor "com.jakewharton:butterknife-compiler:$rootProject.butterknifeVersion"
-    implementation 'com.github.bumptech.glide:glide:3.7.0'
+    implementation "com.github.bumptech.glide:glide:$rootProject.glideVersion"
+    implementation group: 'com.github.bumptech.glide', name: 'gifencoder-integration', version: "$rootProject.glideVersion"
```

Comment:
Why are you writing it this way?

Reply:
Because gifencoder class was not available in the glide  dependency, I had to import that externally by mentioning the dependency and the name.

Reference
https://mvnrepository.com/artifact/com.github.bumptech.glide/gifencoder-integration/4.9.0

Reply:
So, why are you writing it this way?

Reply:
Maven follows this way of adding dependency :P

Reply:
@iamareebjamal Check and merge